e2e-kind.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly KIND_VERSION=0.1.0
+readonly CLUSTER_NAME=chart-testing
+readonly K8S_VERSION=v1.13.2
+
+create_kind_cluster() {
+    kind create cluster --name "$CLUSTER_NAME" --config examples/kind/test/kind-config.yaml --image "kindest/node:$K8S_VERSION" --wait 10s
```

Comment:
I would not use the example config for kind. It's unnecessary as we don't need a multi-mode cluster. I wouldn't rely on examples anyways. They may change independently and break things.

```suggestion
    kind create cluster --name "$CLUSTER_NAME" --image "kindest/node:$K8S_VERSION" --wait 10s
```

Reply:
> They may change independently and break things.

That was why I went with the example config, I wanted to make sure we provide working examples. 😄 I'm happy to remove this though.